### ISSUE TITLE
Help with node red integration.

### ISSUE THREAD

**Issue by marco-dmaa** [davidm-glitch/home-assistant-eheim-digital#30]
Hello, first of all thank you for your fantastic work. 

I'm trying to integrate my filter in Node-RED. As soon as I connect to the websocket I receive all the values. 
But then I continuously receive the "REQ_KEEP_ALIVE" message. 

How can I keep the socket open and continue receiving values from the filter? 

I tried sending this message "{"title": "GET_FILTER_DATA", "to": mac_address, "from": "USER"}" but the filter doesn't respond. 

Could you please help me? 

I know it's not very related to Home Assistant but I'm asking for your help. 

Thank you very much, Marco.

**Comment by davidm-glitch** [comment]
Hey,

If you get the keep alive message it seems that the websocket is still open. If there are any changes from the filter they should be there. 

The message you send, did you replace the Mac address with the actual one of the filter?

**Comment by marco-dmaa** [comment]
Yes, the WebSocket connection remains open but I'm not receiving any values from the filter. 
I tried changing the flow rate and turning the filter on/off from its web page, but no messages are coming through the WebSocket except for the Keep Alive.
In the message i sent the MAC is in format 11:22:33:44:55:66. Is correct?


Thank you, Marco.
